website/app/layout.tsx

```diff
@@ -21,8 +21,12 @@ const inter = Inter({
 });
 
 export const metadata: Metadata = {
-  title: "Create Next App",
-  description: "Generated by create next app",
+  title: "Kardinal",
+  description: "The lightest-weight k8s dev environments in the world",
+  metadataBase: new URL("https://kardinal.dev"),
+  alternates: {
+    canonical: "./",
```

Comment:
the value `./` resolves to the `metadataBase` + the current page path when generated. See https://github.com/vercel/next.js/discussions/50189